libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -474,7 +474,7 @@ void ContractLevelChecker::checkBaseABICompatibility(ContractDefinition const& _
 
 		auto const& currentLoc = func.second->declaration().location();
 
-		for (TypePointer const& paramType: func.second->parameterTypes() + func.second->parameterTypes())
```

Comment:
Was this a typo before?

Reply:
Looks like it's been there from the beginning: 9919670ddd520c3d6fdc1ce1400c081cc6eb7fe4